.gitignore

```diff
@@ -0,0 +1 @@
+server/.vscode/settings.json
```

Comment:
we should add this vscode setup to gitignore

Reply:
Ah, how did this cam up. My bad I ll correct it

Reply:
Hey @LakshyaKhatri  it's in gitignore only.
It is showing the changes made in gitignore

Reply:
oh! sorry, my bad.